### ISSUE TITLE
Cannot install new board managers

### ISSUE THREAD

**Issue by angrest** [arduino/arduino-cli#1959]
### Describe the problem

An existing [github-action](https://github.com/angrest/pocketBME280/blob/3586c5073140b33de5eeb75f83e2775013d62fe2/.github/workflows/compileSketchesMatrix.yml) (not used for a month) now breaks if I try to install another platform:
```
Run arduino-cli config init
[2](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:2)
  arduino-cli config init
[3](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:3)
  arduino-cli core update-index
[4](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:4)
  arduino-cli core upgrade
[5](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:5)
  arduino-cli core install ATTinyCore:avr  --additional-urls http://drazzy.com/package_drazzy.com_index.json
[6](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:6)
  shell: /usr/bin/bash -e {0}
[7](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:7)
  env:
[8](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:8)
    REPO_PATH: libraries/pocketBME280
[9](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:10)
Config file written to: /home/runner/.arduino15/arduino-cli.yaml
[10](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:11)

[11](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:12)
Downloading index: library_index.tar.bz2 0 B / 2.15 MiB    0.00%
[12](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:13)
Downloading index: library_index.tar.bz2 downloaded
[13](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:14)

[14](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:15)
Downloading index: package_index.tar.bz2 0 B / 43.47 KiB    0.00%
[15](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:16)
Downloading index: package_index.tar.bz2 downloaded
[16](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:17)

[17](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:18)
Downloading index: package_index.tar.bz2 0 B / 43.47 KiB    0.00%
[18](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:19)
Downloading index: package_index.tar.bz2 downloaded
[19](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:20)
Downloading missing tool builtin:ctags@5.8-arduino11...
[20](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:21)

[21](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:22)
builtin:ctags@5.8-arduino11 0 B / 108.99 KiB    0.00%
[22](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:23)
builtin:ctags@5.8-arduino11 downloaded
[23](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:24)
Installing builtin:ctags@5.8-arduino11...
[24](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:25)
builtin:ctags@5.8-arduino11 installed
[25](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:26)
Downloading missing tool builtin:serial-discovery@1.3.2...
[26](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:27)

[27](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:28)
builtin:serial-discovery@1.3.2 downloaded
[28](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:29)
Installing builtin:serial-discovery@1.3.2...
[29](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:30)
builtin:serial-discovery@1.3.2 installed
[30](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:31)
Downloading missing tool builtin:mdns-discovery@1.0.6...
[31](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:32)

[32](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:33)
builtin:mdns-discovery@1.0.6 downloaded
[33](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:34)
Installing builtin:mdns-discovery@1.0.6...
[34](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:35)
builtin:mdns-discovery@1.0.6 installed
[35](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:36)
Downloading missing tool builtin:serial-monitor@0.10.0...
[36](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:37)

[37](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:38)
builtin:serial-monitor@0.10.0 downloaded
[38](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:39)
Installing builtin:serial-monitor@0.10.0...
[39](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:40)
builtin:serial-monitor@0.10.0 installed
[40](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:41)
All the cores are already at the latest version
[41](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:42)
Error initializing instance: Loading index file: loading json index file /home/runner/.arduino15/package_drazzy.com_index.json: open /home/runner/.arduino15/package_drazzy.com_index.json: no such file or directory
[42](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:43)
Error initializing instance: Loading index file: loading json index file /home/runner/.arduino15/package_drazzy.com_index.json: open /home/runner/.arduino15/package_drazzy.com_index.json: no such file or directory
[43](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:44)
Invalid argument passed: Found 0 platform for reference "ATTinyCore:avr":
[44](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:45)

[45](https://github.com/angrest/pocketBME280/actions/runs/3370241682/jobs/5590895387#step:4:46)
Error: Process completed with exit code 7.
```


### To reproduce

I can reproduce this in a fresh ubuntu docker container:
```
host:~:# docker run --name ubuntu-latest -h ubuntu-latest -e LANG=C.UTF-8 -it ubuntu /bin/bash -l
root@ubuntu-latest:/# apt update
<output truncated>
All packages are up to date.
root@ubuntu-latest:/# apt -y install curl
<output truncated>
done.
root@ubuntu-latest:/# curl -fsSL https://raw.githubusercontent.com/arduino/arduino-cli/master/install.sh | sh -s nightly-latest
Installing in //bin
ARCH=ARMv7
OS=Linux
Using curl as download tool
Downloading https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-latest_Linux_ARMv7.tar.gz
arduino-cli  Version: nightly-20221025 Commit: f239754 Date: 2022-10-25T01:35:48Z installed successfully in //bin
root@ubuntu-latest:/# arduino-cli -v core install ATTinyCore:avr  --additional-urls http://drazzy.com/package_drazzy.com_index.json
INFO[0000] Config file not found, using default values  
INFO[0000] arduino-cli version nightly-20221025         
INFO[0000] Updating libraries index                     
Downloading index: library_index.tar.bz2 downloaded                                                                                                                                                                                               
INFO[0003] Updating index                                url="https://downloads.arduino.cc/packages/package_index.tar.bz2"
Downloading index: package_index.tar.bz2 downloaded                                                                                                                                                                                               
INFO[0004] Loading hardware from: /root/.arduino15/packages 
INFO[0004] Checking signature                            index=/root/.arduino15/package_index.json signatureFile=/root/.arduino15/package_index.json.sig trusted=true
Error initializing instance: Loading index file: loading json index file /root/.arduino15/package_drazzy.com_index.json: open /root/.arduino15/package_drazzy.com_index.json: no such file or directory
ERRO[0004] Error initializing instance: Loading index file: loading json index file /root/.arduino15/package_drazzy.com_index.json: open /root/.arduino15/package_drazzy.com_index.json: no such file or directory 
Downloading missing tool builtin:mdns-discovery@1.0.6...
builtin:mdns-discovery@1.0.6 downloaded                                                                                                                                                                                                           
INFO[0005] Installing tool                               Tool="builtin:mdns-discovery@1.0.6"
Installing builtin:mdns-discovery@1.0.6...
INFO[0005] Tool installed                                Tool="builtin:mdns-discovery@1.0.6"
builtin:mdns-discovery@1.0.6 installed
Downloading missing tool builtin:serial-monitor@0.10.0...
builtin:serial-monitor@0.10.0 downloaded                                                                                                                                                                                                          
INFO[0006] Installing tool                               Tool="builtin:serial-monitor@0.10.0"
Installing builtin:serial-monitor@0.10.0...
INFO[0006] Tool installed                                Tool="builtin:serial-monitor@0.10.0"
builtin:serial-monitor@0.10.0 installed
Downloading missing tool builtin:ctags@5.8-arduino11...
builtin:ctags@5.8-arduino11 downloaded                                                                                                                                                                                                            
INFO[0006] Installing tool                               Tool="builtin:ctags@5.8-arduino11"
Installing builtin:ctags@5.8-arduino11...
INFO[0006] Tool installed                                Tool="builtin:ctags@5.8-arduino11"
builtin:ctags@5.8-arduino11 installed
Downloading missing tool builtin:serial-discovery@1.3.2...
builtin:serial-discovery@1.3.2 downloaded                                                                                                                                                                                                         
INFO[0006] Installing tool                               Tool="builtin:serial-discovery@1.3.2"
Installing builtin:serial-discovery@1.3.2...
INFO[0007] Tool installed                                Tool="builtin:serial-discovery@1.3.2"
builtin:serial-discovery@1.3.2 installed
INFO[0007] Loading tools from dir: /root/.arduino15/packages/builtin/tools 
INFO[0007] Loaded tool                                   tool="builtin:ctags@5.8-arduino11"
INFO[0007] Loaded tool                                   tool="builtin:mdns-discovery@1.0.6"
INFO[0007] Loaded tool                                   tool="builtin:serial-discovery@1.3.2"
INFO[0007] Loaded tool                                   tool="builtin:serial-monitor@0.10.0"
INFO[0007] Loading libraries index file                  index=/root/.arduino15/library_index.json
INFO[0008] Adding libraries dir                          dir=/root/Arduino/libraries location=user
INFO[0008] Executing `arduino-cli core install`         
INFO[0008] Parsing reference ATTinyCore:avr             
INFO[0008] Loading hardware from: /root/.arduino15/packages 
INFO[0008] Loading package builtin from: /root/.arduino15/packages/builtin 
INFO[0008] Checking existence of 'tools' path: /root/.arduino15/packages/builtin/tools 
INFO[0008] Loading tools from dir: /root/.arduino15/packages/builtin/tools 
INFO[0008] Loaded tool                                   tool="builtin:ctags@5.8-arduino11"
INFO[0008] Loaded tool                                   tool="builtin:mdns-discovery@1.0.6"
INFO[0008] Loaded tool                                   tool="builtin:serial-discovery@1.3.2"
INFO[0008] Loaded tool                                   tool="builtin:serial-monitor@0.10.0"
INFO[0008] Checking signature                            index=/root/.arduino15/package_index.json signatureFile=/root/.arduino15/package_index.json.sig trusted=true
Error initializing instance: Loading index file: loading json index file /root/.arduino15/package_drazzy.com_index.json: open /root/.arduino15/package_drazzy.com_index.json: no such file or directory
ERRO[0008] Error initializing instance: Loading index file: loading json index file /root/.arduino15/package_drazzy.com_index.json: open /root/.arduino15/package_drazzy.com_index.json: no such file or directory 
INFO[0008] Loading libraries index file                  index=/root/.arduino15/library_index.json
INFO[0009] Adding libraries dir                          dir=/root/Arduino/libraries location=user
Invalid argument passed: Found 0 platform for reference "ATTinyCore:avr":

ERRO[0009] Invalid argument passed: Found 0 platform for reference "ATTinyCore:avr": 
```
If I manually download the JSON file and put it into `.arduino`, it does work:
```
root@ubuntu-latest:/# curl http://drazzy.com/package_drazzy.com_index.json -o ~/.arduino15/package_drazzy.com_index.json
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  207k  100  207k    0     0   347k      0 --:--:-- --:--:-- --:--:--  348k
root@ubuntu-latest:/# arduino-cli -v core install ATTinyCore:avr  --additional-urls http://drazzy.com/package_drazzy.com_index.json
INFO[0000] Config file not found, using default values  
INFO[0000] arduino-cli version nightly-20221025         
INFO[0000] Loading hardware from: /root/.arduino15/packages 
INFO[0000] Loading package builtin from: /root/.arduino15/packages/builtin 
INFO[0000] Checking existence of 'tools' path: /root/.arduino15/packages/builtin/tools 
INFO[0000] Loading tools from dir: /root/.arduino15/packages/builtin/tools 
INFO[0000] Loaded tool                                   tool="builtin:ctags@5.8-arduino11"
INFO[0000] Loaded tool                                   tool="builtin:mdns-discovery@1.0.6"
INFO[0000] Loaded tool                                   tool="builtin:serial-discovery@1.3.2"
INFO[0000] Loaded tool                                   tool="builtin:serial-monitor@0.10.0"
INFO[0000] Checking signature                            index=/root/.arduino15/package_index.json signatureFile=/root/.arduino15/package_index.json.sig trusted=true
INFO[0000] Checking signature                            error="opening signature file: open /root/.arduino15/package_drazzy.com_index.json.sig: no such file or directory" index=/root/.arduino15/package_drazzy.com_index.json signatureFile=/root/.arduino15/package_drazzy.com_index.json.sig
INFO[0000] Loading libraries index file                  index=/root/.arduino15/library_index.json
INFO[0001] Adding libraries dir                          dir=/root/Arduino/libraries location=user
INFO[0001] Executing `arduino-cli core install`         
INFO[0001] Parsing reference ATTinyCore:avr             
INFO[0001] Loading hardware from: /root/.arduino15/packages 
INFO[0001] Loading package builtin from: /root/.arduino15/packages/builtin 
INFO[0001] Checking existence of 'tools' path: /root/.arduino15/packages/builtin/tools 
INFO[0001] Loading tools from dir: /root/.arduino15/packages/builtin/tools 
INFO[0001] Loaded tool                                   tool="builtin:ctags@5.8-arduino11"
INFO[0001] Loaded tool                                   tool="builtin:mdns-discovery@1.0.6"
INFO[0001] Loaded tool                                   tool="builtin:serial-discovery@1.3.2"
INFO[0001] Loaded tool                                   tool="builtin:serial-monitor@0.10.0"
INFO[0001] Checking signature                            index=/root/.arduino15/package_index.json signatureFile=/root/.arduino15/package_index.json.sig trusted=true
INFO[0001] Checking signature                            error="opening signature file: open /root/.arduino15/package_drazzy.com_index.json.sig: no such file or directory" index=/root/.arduino15/package_drazzy.com_index.json signatureFile=/root/.arduino15/package_drazzy.com_index.json.sig
INFO[0001] Loading libraries index file                  index=/root/.arduino15/library_index.json
INFO[0002] Adding libraries dir                          dir=/root/Arduino/libraries location=user
INFO[0002] Running from console, will run post-install by default 
Downloading packages...
arduino:avr-gcc@7.3.0-atmel3.6.1-arduino7 downloaded                                                                                                                                                                                              
arduino:avrdude@6.3.0-arduino18 downloaded                                                                                                                                                                                                        
ATTinyCore:micronucleus@2.5-azd1b downloaded                                                                                                                                                                                                      
ATTinyCore:avr@1.5.2 downloaded                                                                                                                                                                                                                   
INFO[0009] Installing tool                               Tool="arduino:avr-gcc@7.3.0-atmel3.6.1-arduino7"
Installing arduino:avr-gcc@7.3.0-atmel3.6.1-arduino7...
INFO[0039] Tool installed                                Tool="arduino:avr-gcc@7.3.0-atmel3.6.1-arduino7"
arduino:avr-gcc@7.3.0-atmel3.6.1-arduino7 installed
INFO[0039] Installing tool                               Tool="arduino:avrdude@6.3.0-arduino18"
Installing arduino:avrdude@6.3.0-arduino18...
INFO[0039] Tool installed                                Tool="arduino:avrdude@6.3.0-arduino18"
arduino:avrdude@6.3.0-arduino18 installed
INFO[0039] Installing tool                               Tool="ATTinyCore:micronucleus@2.5-azd1b"
Installing ATTinyCore:micronucleus@2.5-azd1b...
INFO[0039] Tool installed                                Tool="ATTinyCore:micronucleus@2.5-azd1b"
ATTinyCore:micronucleus@2.5-azd1b installed
INFO[0039] Installing platform                           platform="ATTinyCore:avr@1.5.2"
Installing platform ATTinyCore:avr@1.5.2...
INFO[0040] Running post_install script                   platform="ATTinyCore:avr@1.5.2"
Configuring platform....
INFO[0040] Platform installed                            platform="ATTinyCore:avr@1.5.2"
Platform ATTinyCore:avr@1.5.2 installed
INFO[0040] Loading hardware from: /root/.arduino15/packages 
INFO[0040] Loading package ATTinyCore from: /root/.arduino15/packages/ATTinyCore/hardware 
INFO[0040] Checking signature                            error="opening signature file: open /root/.arduino15/packages/ATTinyCore/hardware/avr/1.5.2/installed.json.sig: no such file or directory" index=/root/.arduino15/packages/ATTinyCore/hardware/avr/1.5.2/installed.json signatureFile=/root/.arduino15/packages/ATTinyCore/hardware/avr/1.5.2/installed.json.sig
INFO[0040] Adding monitor tool                           protocol=serial tool="builtin:serial-monitor"
INFO[0040] Loaded platform                               platform="ATTinyCore:avr@1.5.2"
INFO[0040] Checking existence of 'tools' path: /root/.arduino15/packages/ATTinyCore/tools 
INFO[0040] Loading tools from dir: /root/.arduino15/packages/ATTinyCore/tools 
INFO[0040] Loaded tool                                   tool="ATTinyCore:micronucleus@2.5-azd1b"
INFO[0040] Loading package arduino from: /root/.arduino15/packages/arduino 
INFO[0040] Checking existence of 'tools' path: /root/.arduino15/packages/arduino/tools 
INFO[0040] Loading tools from dir: /root/.arduino15/packages/arduino/tools 
INFO[0040] Loaded tool                                   tool="arduino:avr-gcc@7.3.0-atmel3.6.1-arduino7"
INFO[0040] Loaded tool                                   tool="arduino:avrdude@6.3.0-arduino18"
INFO[0040] Loading package builtin from: /root/.arduino15/packages/builtin 
INFO[0040] Checking existence of 'tools' path: /root/.arduino15/packages/builtin/tools 
INFO[0040] Loading tools from dir: /root/.arduino15/packages/builtin/tools 
INFO[0040] Loaded tool                                   tool="builtin:ctags@5.8-arduino11"
INFO[0040] Loaded tool                                   tool="builtin:mdns-discovery@1.0.6"
INFO[0040] Loaded tool                                   tool="builtin:serial-discovery@1.3.2"
INFO[0040] Loaded tool                                   tool="builtin:serial-monitor@0.10.0"
INFO[0040] Checking signature                            index=/root/.arduino15/package_index.json signatureFile=/root/.arduino15/package_index.json.sig trusted=true
INFO[0040] Checking signature                            error="opening signature file: open /root/.arduino15/package_drazzy.com_index.json.sig: no such file or directory" index=/root/.arduino15/package_drazzy.com_index.json signatureFile=/root/.arduino15/package_drazzy.com_index.json.sig
INFO[0040] Adding libraries dir                          dir=/root/.arduino15/packages/ATTinyCore/hardware/avr/1.5.2/libraries location=platform
INFO[0040] Loading libraries index file                  index=/root/.arduino15/library_index.json
INFO[0041] Adding libraries dir                          dir=/root/Arduino/libraries location=user
```

### Expected behavior

board manager is installed without errors.

### Arduino CLI version

nightly-20221025 

### Operating system

Linux

### Operating system version

Linux ubuntu-latest 5.4.199-odroidxu4 #22.05.3 SMP PREEMPT Wed Jun 22 07:29:40 UTC 2022 armv7l armv7l armv7l GNU/Linux

### Additional context

This is reproducible with both the ATTTiny board manager from the example above as well as with the ESP32 board manager `https://raw.githubusercontent.com/espressif/arduino-esp32/gh-pages/package_esp32_index.json`

### Issue checklist

- [X] I searched for previous reports in [the issue tracker](https://github.com/arduino/arduino-cli/issues?q=)
- [X] I verified the problem still occurs when using the [nightly build](https://arduino.github.io/arduino-cli/dev/installation/#nightly-builds)
- [X] My report contains all necessary details

**Comment by per1234** [comment]
Hi @angrest. You must run the `arduino-cli core update-index` command before `core install`.

**Comment by angrest** [comment]
did not solve the problem.
freshly created docker container as above
```
root@ubuntu-latest:/# arduino-cli config init
Config file written to: /root/.arduino15/arduino-cli.yaml
root@ubuntu-latest:/# arduino-cli core update-index
Downloading index: library_index.tar.bz2 downloaded                             
Downloading index: package_index.tar.bz2 downloaded                             
Downloading index: package_index.tar.bz2 downloaded                             
root@ubuntu-latest:/# arduino-cli core install esp32:esp32 --additional-urls https://raw.githubusercontent.com/espressif/arduino-esp32/gh-pages/package_esp32_index.json
Downloading missing tool builtin:ctags@5.8-arduino11...
builtin:ctags@5.8-arduino11 downloaded                                          
Installing builtin:ctags@5.8-arduino11...
builtin:ctags@5.8-arduino11 installed
Downloading missing tool builtin:serial-discovery@1.3.2...
builtin:serial-discovery@1.3.2 downloaded                                       
Installing builtin:serial-discovery@1.3.2...
builtin:serial-discovery@1.3.2 installed
Downloading missing tool builtin:mdns-discovery@1.0.6...
builtin:mdns-discovery@1.0.6 downloaded                                         
Installing builtin:mdns-discovery@1.0.6...
builtin:mdns-discovery@1.0.6 installed
Downloading missing tool builtin:serial-monitor@0.10.0...
builtin:serial-monitor@0.10.0 downloaded                                        
Installing builtin:serial-monitor@0.10.0...
builtin:serial-monitor@0.10.0 installed
Error initializing instance: Loading index file: loading json index file /root/.arduino15/package_esp32_index.json: open /root/.arduino15/package_esp32_index.json: no such file or directory
Error initializing instance: Loading index file: loading json index file /root/.arduino15/package_esp32_index.json: open /root/.arduino15/package_esp32_index.json: no such file or directory
Invalid argument passed: Found 0 platform for reference "esp32:esp32":
```

**Comment by per1234** [comment]
If you are going to specify package index URLs via the `--additional-urls` flag, you must do that in every command that relies on that information. `core update-index` very much relies on that information.

If you need additional assistance using Arduino CLI, we are always happy to help out over on its dedicated category in the Arduino forum:

https://forum.arduino.cc/c/software/arduino-cli/89

**Comment by angrest** [comment]
as written above, the action worked without any problem four weeks ago (it is actually based on your contribution 😊 ) - so, why should it stop working suddenly? The only thing which changed meanwhile is arduino-cli was updated to 0.28.0

**Comment by angrest** [comment]
I've meanwhile tried v0.27.0 - there it works without problem (with and without previous `core update-index`). 
Fresh container:
```
root@ubuntu-latest:/# curl -fsSL https://raw.githubusercontent.com/arduino/arduino-cli/master/install.sh | sh -s 0.27.0
Installing in //bin
ARCH=ARMv7
OS=Linux
Using curl as download tool
Downloading https://downloads.arduino.cc/arduino-cli/arduino-cli_0.27.0_Linux_ARMv7.tar.gz


A new release of Arduino CLI is available: 0.27.0 → 0.28.0
https://arduino.github.io/arduino-cli/latest/installation/#latest-packages
arduino-cli  Version: 0.27.0 Commit: c2af7c5a Date: 2022-09-05T08:10:30Z installed successfully in //bin
root@ubuntu-latest:/# arduino-cli config init
Config file written to: /root/.arduino15/arduino-cli.yaml
root@ubuntu-latest:/# arduino-cli core install esp32:esp32 --additional-urls https://raw.githubusercontent.com/espressif/arduino-esp32/gh-pages/package_esp32_index.json
Downloading index: library_index.json.gz downloaded                             
Downloading index signature: library_index.json.sig downloaded                  
<output truncated>
Platform esp32:esp32@2.0.5 installed
root@ubuntu-latest:/#
